### PR TITLE
add 3 code examples

### PR DESCRIPTION
**Pull request by xflesym**
add_google_my_business_location_extensions.rb
get_campaign_criterion_bid_modifier_simulations.rb
add_shopping_smart_ad.rb

### CODE REVIEW

**Comment by xflesym** [comment]
Friendly ping for review

File: examples/extensions/add_google_my_business_location_extensions.rb [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    feed.name = "Google My Business feed #{(Time.new.to_f * 1000).to_i}"
+    feed.origin = :GOOGLE
+    feed.places_location_feed_data = client.resource.places_location_feed_data do |data|
+      data.email_address = gmb_email_address
+      data.business_account_id = business_account_identifier
+      data.label_filters << "Stores in New York"
+      data.oauth_info = client.resource.o_auth_info do |oauth|
+        oauth.http_method = "GET"
+        oauth.http_request_url = "https://www.googleapis.com/auth/adwords"
+        oauth.http_authorization_header = "Bearer #{gmb_access_token}"
```
Comment (mcloonan) [inline examples/extensions/add_google_my_business_location_extensions.rb:62]: I think we need to scrub this field in our logging implementation. Can you check with Anash?

File: examples/shopping_ads/add_shopping_smart_ad.rb [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    # A product group is a subset of inventory. Listing groups are the equivalent
+    # of product groups in the API and allow you to bid on the chosen group or
+    # exclude a group from bidding.
+    # This method creates an ad group criterion containing a listing group.
+    add_default_shopping_listing_group(client, customer_id, ad_group_name)
+  end
+end
+
+def add_campaign_budget(client, customer_id)
+
```
Comment (mcloonan) [inline examples/shopping_ads/add_shopping_smart_ad.rb:72]: nit: remove blank line at start of function

File: examples/shopping_ads/add_shopping_smart_ad.rb [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    # http://support.google.com/google-ads/answer/7684216.
+    campaign.maximize_conversion_value = client.resource.maximize_conversion_value do |m|
+      m.target_roas = 3.5
+    end
+
+    campaign.campaign_budget = budget_name
+  end
+
+  service = client.service.campaign
+  response = service.mutate_campaigns(
```
Comment (mcloonan) [inline examples/shopping_ads/add_shopping_smart_ad.rb:135]: Just make the service inline if you only use it once: response = client.service.campaign.mutate_campaigns(...)

**Review by mcloonan — CHANGES_REQUESTED** [review]
Devin is OoO, so please choose a different reviewer.

File: examples/shopping_ads/add_shopping_smart_ad.rb (hunk 2 above)
Comment (xflesym) [reply examples/shopping_ads/add_shopping_smart_ad.rb]: done.

File: examples/shopping_ads/add_shopping_smart_ad.rb (hunk 3 above)
Comment (xflesym) [reply examples/shopping_ads/add_shopping_smart_ad.rb]: done.

File: examples/extensions/add_google_my_business_location_extensions.rb (hunk 1 above)
Comment (xflesym) [reply examples/extensions/add_google_my_business_location_extensions.rb:62]: @AnashOommen 
Hi Anash - Want to confirm with you, for this `oauth.http_authorization_header` field, we need to scrub it in our logging, right?

Comment (AnashOommen) [reply examples/extensions/add_google_my_business_location_extensions.rb:62]: It looks like a candidate for scrubbing, but IIRC we don't do it today. Could you add a frontlog entry and possibly a followup CL to add scrubbing?

**Review by AnashOommen — APPROVED** [review]

**Review by mcloonan — APPROVED** [review]

File: examples/extensions/add_google_my_business_location_extensions.rb (hunk 1 above)
Comment (xflesym) [reply examples/extensions/add_google_my_business_location_extensions.rb:62]: Sure. I've added to our client lib front log. Will revisit and add a followup CL.

**Pull request merged**